plugin.rb

```diff
@@ -38,6 +38,24 @@ module ::DiscourseTranslator
     end
   end
 
+  on(:post_process_cooked) do |_, post|
+    if SiteSetting.automatic_translation_target_languages.present?
+      Jobs.enqueue(:translate_translatable, type: Post, translatable_id: post.id)
```

Comment:
Nit pick but because in the job you are using `args[:type].constantize` maybe we can pass a string like `type: "Post"` instead of passing the whole object.